src/components/ContactEdit.tsx

```diff
@@ -113,9 +115,10 @@ interface PropsType {
   collections: EteSync.CollectionInfo[];
   initialCollection?: string;
   item?: ContactType;
-  onSave: (contact: ContactType, journalUid: string, originalContact?: ContactType) => void;
+  onSave: (contact: ContactType, journalUid: string, originalContact?: ContactType) => Promise<any>;
```

Comment:
Same here with `Promise<void>` - same with all of these places, I won't comment for each.